nanomq/apps/broker.c

```diff
@@ -1632,12 +1632,14 @@ broker_start(int argc, char **argv)
 	return rc == 0 ? EXIT_SUCCESS : EXIT_FAILURE;
 }
 
-int
-broker_start_with_conf(conf *nanomq_conf)
+void *
+broker_start_with_conf(void *nmq_conf)
 {
 
 	int rc = 0;
 	int pid = 0;
+	int *res = &rc;
+	conf *nanomq_conf = nmq_conf;
```

Comment:
Setting rc and res at the same time is very confusing, both do the same thing

Reply:
yes u r right.